### DISCUSSION TITLE
How to handle one-to-many?

### DISCUSSION THREAD

**Discussion by rxndxm** [TomGrozev/arangox_ecto#38] [Q&A] [answered]
👋🏼 If I have two schemas: Teacher and Student

I see `one_incoming :teacher, MyApp.Teacher` goes in the Student schema, but I don't see an equivalent for the Teacher schema

`outgoing` creates an edge collection, and `one_outgoing` doesn't seem to handle the `has_many` case?

**Comment by TomGrozev** [comment, marked as answer]
Since ArangoDB is not relational database, it does not work like this. The `one_outgoing` and `one_incoming` store the `_id` of the document in the field so it cannot be many, only one. If you want to do any one-to-many or many-to-many operations in ArangoDB you have to use edge collections. So this is the correct approach.
